src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -190,6 +223,7 @@ public void UpdateDescriptionForModel(MappingDataFormatParameter mappingSource)
             {
                column.ErrorStdDev = Constants.STD_DEV_GEOMETRIC;
                column.Unit = new UnitDescription(""); //geometric error has no unit
```

Comment:
same stuff here. Use the dimension base unit

Reply:
Fixed